docs/csharp/language-reference/compiler-options/codepage-compiler-option.md

```diff
@@ -34,7 +34,7 @@ This option specifies which codepage to use during compilation if the required p
   
  If the source code files were created with the same codepage that is in effect on your computer or if the source code files were created with UNICODE or UTF-8, you need not use **/codepage**.  
   
- See [GetCPInfo](http://go.microsoft.com/fwlink/?LinkId=148371) for information on how to find which code pages are supported on your system.  
+ See [GetCPInfo](https://msdn.microsoft.com/en-us/library/dd318078(VS.85).aspx) for information on how to find which code pages are supported on your system.  
```

Comment:
just remove the locale from the link 

https://msdn.microsoft.com/library/dd318078(VS.85).aspx